frontend/src/components/Admin/DevPortfolio/DevPortfolioDetails.tsx

```diff
@@ -176,10 +188,38 @@ const SubmissionDetails: React.FC<SubmissionDetailsProps> = ({ submission, isAdm
         />
       </Table.Cell>
       {isAdminView ? (
-        <Table.Cell rowSpan={`${numRows}`}>{isValid ? 'Valid' : 'Invalid'}</Table.Cell>
+        <Table.Cell rowSpan={`${numRows}`}>
+          {<div>{(hasText && 'Pending') || (isValid ? 'Valid' : 'Invalid')}</div>}
+        </Table.Cell>
       ) : (
         <></>
       )}
+      {submission.text ? (
+        <Table.Cell rowSpan={`${numRows}`}>
+          <Modal
+            onClose={() => setViewText(false)}
+            onOpen={() => setViewText(true)}
+            open={viewText}
+            trigger={<Button>Show Text</Button>}
+          >
+            <Modal.Header>
+              Paragraph Response for {submission.member.firstName} {submission.member.lastName}
+            </Modal.Header>
+            <Modal.Content image>
+              <Modal.Description>
+                <p>{submission.text}</p>
+              </Modal.Description>
+            </Modal.Content>
+            <Modal.Actions>
+              <Button color="red" onClick={() => setViewText(false)}>
+                Close
+              </Button>
+            </Modal.Actions>
+          </Modal>
+        </Table.Cell>
```

Comment:
Thoughts on moving the modal to `components/Modals/`?

Reply:
Yea that's a good idea, especially since there are going to be other features in the future for the dev portfolios where the admins are going to view text as part of the response.